.storybook/preview.js

```diff
@@ -16,3 +16,15 @@ export const decorators = [
     </ThemeProvider>
   )
 ];
+
+export const parameters = {
+  docs: {
+    extractComponentDescription: (_component, { notes }) => {
+      if (notes) {
+        return typeof notes === 'string' ? notes : notes.markdown || notes.text;
+      }
+
+      return null;
```

Comment:
This is the recommended config when migrating from `notes` to `docs`.  This pulls our existing notes into the Docs pages.